### PR TITLE
#2431 Initial release notes for 0.7.2

### PR DESCRIPTION
**Pull request by johannes-b**

### CODE REVIEW

File: releasenotes/releasenotes_V0.7.2.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Release Notes 0.7.2
+
+Keptn 0.7.2 improves the capabilities of the 0.7.0 release by adding ...
```
Comment (christian-kreuzberger-dtx) [inline releasenotes/releasenotes_V0.7.2.md:3]: replace `0.7.0` with `0.7.1`

File: releasenotes/releasenotes_V0.7.2.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# Release Notes 0.7.2
+
+Keptn 0.7.2 improves the capabilities of the 0.7.0 release by adding ... 
+
+**The key announcements of Keptn 0.7.2**:
+
+:rocket: *Improved UX for Quality Gates*: This release introduces an `/evaluation` endpoint that brings the same user-experience to the API as used from the Keptn CLI. Hence, easily trigger a quality gate evaluation based on a small set of parameters, like: `start`, `end`, and `timeframe`.
```
Comment (christian-kreuzberger-dtx) [inline releasenotes/releasenotes_V0.7.2.md:7]: Should we add here that the provided project, service and stage are also evaluated?

File: releasenotes/releasenotes_V0.7.2.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+</p>
+</details>
+
+<details><summary>CLI</summary>
+<p>
+
+- The output of `keptn version` shows the Keptn API version [#2412](https://github.com/keptn/keptn/issues/2412)
+- Improve robustness and UX of `keptn generate support-archive` [#2340](https://github.com/keptn/keptn/issues/2340)
+- Point user to upgrade docs, fixed bug in `keptn update project` [#2293](https://github.com/keptn/keptn/issues/2293)
```
Comment (christian-kreuzberger-dtx) [inline releasenotes/releasenotes_V0.7.2.md:41]: let's also add https://github.com/keptn/keptn/issues/2445

Comment (johannes-b) [reply releasenotes/releasenotes_V0.7.2.md:42]: 2445 is already contained.

File: releasenotes/releasenotes_V0.7.2.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+**The key announcements of Keptn 0.7.2**:
+
+:rocket: *Improved UX for Quality Gates*: This release introduces an `/evaluation` endpoint that brings the same user-experience to the API as used from the Keptn CLI. Hence, easily trigger a quality gate evaluation based on the parameters `start`, `end`, and `timeframe` for a specific `service` in a particular `project`/`stage`.
+
+:sparkles: *Focused UI enhancements for Quality Gates*: The evaluation tile that displays the quality gate evaluation result has been improved in various ways. First, the Heatmap in this tile now highlights the currently selected evaluation result. Second, it also displays the evaluation results that were used for comparisons giving the user more insights into the quality gate. 
+
+:tada: *Usage statistics for Keptn installation*: If you want to get more insights into your Keptn installation, feel free to deploy the [statistics-service](https://github.com/keptn-sandbox/statistics-service) from the Keptn sandbox. This service provides usage statistics based on events and service executions.
+
```
Comment (christian-kreuzberger-dtx) [inline releasenotes/releasenotes_V0.7.2.md:12]: we should add one more bullet point here regarding the spec change for evaluation-done events.

*Spec update*: sh.keptn.events.evaluation-done cloud-events now have a new field called comparedEvents - see github.com/keptn/spec/... for more details.

File: releasenotes/releasenotes_V0.7.2.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
+# Release Notes 0.7.2
+
+Keptn 0.7.2 enhances the 0.7.1 release with great UX/UI improvements; especially for working with Quality Gates. This release introduces a new API endpoint for triggering a quality gate evaluation and the Bridge shows more details about the evaluation result.
+
+**The key announcements of Keptn 0.7.2**:
+
+:rocket: *Improved UX for Quality Gates*: This release introduces an `/evaluation` endpoint that brings the same user-experience to the API as used from the Keptn CLI. Hence, easily trigger a quality gate evaluation based on the parameters `start`, `end`, and `timeframe` for a specific `service` in a particular `project`/`stage`.
```
Comment (christian-kreuzberger-dtx) [inline releasenotes/releasenotes_V0.7.2.md:7]: Hence, *you can now* easily trigger

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM (please squash merge)

**Pull request merged**